sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/ControlFlowGraph/UcfgMethod.cs

```diff
@@ -99,6 +99,11 @@ public static UcfgMethod Create(IMethodSymbol methodSymbol)
             }
         }
 
+        public static UcfgMethod Create(INamedTypeSymbol typeSymbol)
```

Comment:
[for later] UcfgMethod should become something like `UcfgTypeHelper` with `GetMethodId` and `GetClassName` or to be split into two...

Reply:
Good point... I'll do that now